index.js

```diff
@@ -11,7 +11,12 @@ function Flatiron (inputTree, options) {
   if (!(this instanceof Flatiron)) return new Flatiron(inputTree, options);
 
   this.inputTree = inputTree;
-  this.options = options;
+  this.options = {
+    trimExtensions: Boolean(options.trimExtensions),
+    prefix: options.prefix ? String(options.prefix) : "export default ",
+    suffix: options.suffix ? String(options.suffix) : ';',
```

Comment:
Please use these lines, otherwise it's impossible to pass an empty (`""`) string for no prefix and suffix.

```js
  prefix: options.prefix === undefined ? "export default " : String(options.prefix),
  suffix: options.suffix === undefined ? ";" : String(options.suffix)
```